src/test/kotlin/com/forgerock/sapi/gateway/ob/uk/tests/functional/payment/domestic/standing/order/consents/api/v3_1_8/GetDomesticStandingOrderConsents.kt

```diff
@@ -18,23 +18,14 @@ class GetDomesticStandingOrderConsents(val version: OBVersion, val tppResource:
         // Given
         val consentRequest =
             OBWriteDomesticStandingOrderConsentTestDataFactory.aValidOBWriteDomesticStandingOrderConsent5()
-        val consent =
-            createDomesticStandingOrderConsentsApi.createDomesticStandingOrderConsent(consentRequest)
-
-        assertThat(consent).isNotNull()
-        assertThat(consent.data).isNotNull()
-        assertThat(consent.data.consentId).isNotEmpty()
-        Assertions.assertThat(consent.data.status.toString()).`is`(Status.consentCondition)
-        assertThat(consent.risk).isNotNull()
-
         // When
-        val result = createDomesticStandingOrderConsentsApi.getPatchedConsent(consent)
-
+        val consentResponse =
+            createDomesticStandingOrderConsentsApi.createDomesticStandingOrderConsent(consentRequest)
         // Then
-        assertThat(result).isNotNull()
-        assertThat(result.data).isNotNull()
-        assertThat(result.risk).isNotNull()
-        assertThat(result.data).isEqualTo(consent.data)
-        assertThat(result.risk).isEqualTo(consent.risk)
```

Comment:
Where are these tests done now?

Reply:
Just in the same method, The variables has been changed to consentRequest and consentResponse to be more readable.